src/main/java/org/killbill/billing/client/model/SubscriptionUsage.java

```diff
@@ -0,0 +1,76 @@
+package org.killbill.billing.client.model;
```

Comment:
Sorry Pierre I don't understand exactly what do you mean: for the files we must add copyright header? 
RAT - what does it mean?